portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,31 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+    const facts = [
+        "I don't drink coffee",
+        "My birthday is on 29th of May",
+        "I love vanilla ice cream",
+        "My favourite author is Jane Austen",
+        "I love ice-skating",
+    ];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Pick a random fact.
+    const chosenFact = facts[Math.floor(Math.random() * facts.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+    // Add it to the page.
+    const factContainer = document.getElementById("fact-container");
+    factContainer.innerText = chosenFact;
+}
+
+function addPhoto() {
+    const imgElement = document.createElement('img');
+    // Set the url of the image
+    imgElement.src = 'images/friends.jpg';
+    imgElement.width = "600";
```

Comment:
Try to be consistent in your use of single or double quotes.

Style guide suggests single quotes everywhere: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes